use-cases/vehicle-routing/src/main/java/org/acme/vehiclerouting/solver/justifications/MinimizeTravelTimeJustification.java

```diff
@@ -0,0 +1,15 @@
+package org.acme.vehiclerouting.solver.justifications;
+
+import ai.timefold.solver.core.api.score.stream.ConstraintJustification;
+
+import org.acme.vehiclerouting.domain.Vehicle;
+
+import java.time.Duration;
+
+public record MinimizeTravelTimeJustification(Vehicle vehicle, String description) implements ConstraintJustification {
+
+    public MinimizeTravelTimeJustification(Vehicle vehicle) {
+        this(vehicle, "Vehicle '%s' total travel time is '%s'."
```

Comment:
This is how the result looks like:
"Vehicle '1' total travel time is 'PT5H44M7S'."

While the ISO duration format is nice in API, it's not so user-friendly in the UI.

Option A) format to show hours and minutes
Option B) make the record return only the vehicle (or even vehicleId) and the travel time duration; format on the client

I incline towards B) as it gives the client freedom on how to display or interpret the data.

Reply:
That may be true, but Marek actually just copied this over from the other quickstarts, which already do it like this. Sounds like we're going back and forth and, for sake of consistency, changing _everything_ back and forth.

Reply:
True. AFAIK we have the score analysis only in the School Timetabling QS. 

If we aim for adding it to all QS, now is the time to agree if we keep it or change it. Changing this later will only get more painful.

Reply:
Additional observation - it turns out that having the whole `Vehicle` instance in the Justification is problematic for JSON deserialization because a vehicle references customers that are not part of the ScoreAnalysis. Including them artificially into the Justification could work but is very likely not desired.